tileview/src/main/java/com/qozix/tileview/tiles/TileCanvasViewGroup.java

```diff
@@ -248,12 +249,13 @@ void onRenderTaskPostExecute() {
     if( mTileRenderListener != null ) {
       mTileRenderListener.onRenderComplete();
     }
-    invalidate();
-    requestRender();
```

Comment:
i'd push back here - mostly because i can imagine a scenario where a request render happens, then state is changed in a subtle way that does not start a new render request, but also because it seems a bit out of scope here (more on that later).